sdk/storage/azure-storage-file-datalake/tests/test_directory.py

```diff
@@ -1095,6 +1098,29 @@ def test_get_properties(self, datalake_storage_account_name, datalake_storage_ac
         self.assertIsNotNone(properties.metadata)
         self.assertEqual(properties.metadata['hello'], metadata['hello'])
 
+    @DataLakePreparer()
+    def test_directory_encryption_scope(self, datalake_storage_account_name, datalake_storage_account_key):
+        # Arrange
+        url = self.account_url(datalake_storage_account_name, 'dfs')
+        self.dsc = DataLakeServiceClient(url, credential=datalake_storage_account_key, logging_enable=True)
+        self.config = self.dsc._config
+        self.file_system_name = self.get_resource_name('filesystem')
+        dir_name = 'testdir'
+        file_system = self.dsc.get_file_system_client(self.file_system_name)
+        try:
```

Comment:
I don't think we should have the `try/catch` in here? I know it makes the test more annoying to run but we are explicitly testing this file system being constructed in a specific way (with encryption scope) so it's important this test, as it is, is the one that makes the file system.

Also, I think the test would fail on `create_directory` anyway if this fs was not deleted.

Reply:
Good point, I will make this adjustment.